lint/linter.py

```diff
@@ -1388,7 +1388,7 @@ def split_match(self, match):
         if not match:
             persist.debug('No match for regex: {}'.format(self.regex.pattern))
         else:
-            match_dict.update(match.groupdict())
+            match_dict.update({key: value for key, value in match.groupdict().items() if key in match_dict})
```

Comment:
Could you please wrap that long dict comprehension according to the pattern:
```
[expression
for value in collection
if condition]
```
Readability counts.

Reply:
No problem, this has been updated in the latest commit.